### ISSUE TITLE
fontforge clears clipboard after closing active windows

### ISSUE THREAD

**Issue by aminabedi68** [fontforge/fontforge#4935]
fontforge clears clipboard contents(copied text) after closing the active window, so you have to use another text editing app to paste content and copy from there again.

**Comment by jtanx** [comment]
Yeah I've seen this too, it's a gdk issue. Should be addressed if/when I finish the qt port
